src/uct/ib/rc/accel/rc_mlx5_devx.c

```diff
@@ -117,6 +117,93 @@ uct_rc_mlx5_devx_init_rx_tm(uct_rc_mlx5_iface_common_t *iface,
     return status;
 }
 
+ucs_status_t uct_rc_mlx5_devx_init_rx(uct_rc_mlx5_iface_common_t *iface,
+                                      const uct_rc_iface_common_config_t *config)
+{
+    uct_ib_mlx5_md_t *md  = ucs_derived_of(uct_ib_iface_md(&iface->super.super),
+                                           uct_ib_mlx5_md_t);
+    uct_ib_device_t *dev  = &md->super.dev;
```

Comment:
very similar to `uct_rc_mlx5_devx_init_rx_tm`
looks like can extract common code

Reply:
done